kalite/distributed/backup.py

```diff
@@ -0,0 +1,36 @@
+import dbbackup
+import datetime
+import os
+import sys
+import time
+from os.path import expanduser
```

Comment:
Nah, you should be fine with just `import os`, AFAIK it's the common solution :)